shippable.jobs.yml

```diff
@@ -9,6 +9,8 @@ jobs:
         # manually trigger the cluster provisioning job and not on every commit to the repository
         switch: off
       - IN: gke_cliConfig
+        scopes:
```

Comment:
required to set the gcloud sdk to use the region specified in your `gke_cliConfig` resource's pointer region section